src/data/Sort/Sort.tsx

```diff
@@ -35,6 +35,16 @@ export interface SortProps extends PropsWithChildren {
    * The icon to display
    */
   icon?: any;
+
+  /**
+   * The neutral icon to display.
+   */
+  neutralIcon?: (props: { className?: string }) => React.JSX.Element;
```

Comment:
This type doesn't seem quite right - can you also updated the regular `icon` type as well.

Reply:
ok, hows this?
```
neutralIcon?: React.ComponentType<{ className?: string }>;
```

Reply:
It doesn't have to be just className though. It could be kinda anything - even a emoji if they wanted.

Reply:
the `P` in `React.ComponentType<P>` refers to the components props right? Isn't the only prop that will ever get passed to these icon components going to be `className` based off the usage in here below?

```
 <NeutralIcon
    className={classNames(css.icon, neutralIconClassName)}
/>
```

Reply:
I see what you mean now - i think its fine. LGTM

Reply:
oh shoot I didn't actually push the commit to update to `neutralIcon?: React.ComponentType<{ className?: string }>;` 😅